mrp_multi_level/models/product_mrp_area.py

```diff
@@ -109,6 +109,7 @@ class ProductMRPArea(models.Model):
         string="Group Days of Estimates",
         default=1,
     )
+    to_be_exploded = fields.Boolean(compute="compute_to_be_exploded")
```

Comment:
no default implementation for `compute_to_be_exploded`? also, it should be a private method, could you add the underscore at the begging (`_compute_to_be_exploded`)

Reply:
Oops did not commit all the changes. Thanks for letting me know.